libcomps/src/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+include( GNUInstallDirs )
```

Comment:
Can you please drop the spaces inside the brackets to be consistent with the rest of the file?

The patch works now looks good otherwise.